app/celery/nightly_tasks.py

```diff
@@ -252,70 +251,6 @@ def get_letter_notifications_still_sending_when_they_shouldnt_be():
     return q.count(), expected_sent_date
 
 
-@notify_celery.task(name="raise-alert-if-no-letter-ack-file")
-@cronitor("raise-alert-if-no-letter-ack-file")
-def letter_raise_alert_if_no_ack_file_for_zip():
-    # get a list of zip files since yesterday
-    zip_file_set = set()
-    today_str = datetime.utcnow().strftime("%Y-%m-%d")
-    yesterday = datetime.now(tz=pytz.utc) - timedelta(days=1)  # AWS datetime format
-
-    for key in s3.get_list_of_files_by_suffix(
-        bucket_name=current_app.config["S3_BUCKET_LETTERS_PDF"], subfolder=today_str + "/zips_sent", suffix=".TXT"
-    ):
-        subname = key.split("/")[-1]  # strip subfolder in name
-        zip_file_set.add(subname.upper().replace(".ZIP.TXT", ""))
-
-    # get acknowledgement file
-    ack_file_set = set()
-
-    for key in s3.get_list_of_files_by_suffix(
-        bucket_name=current_app.config["S3_BUCKET_DVLA_RESPONSE"],
-        subfolder="root/dispatch",
-        suffix=".ACK.txt",
-        last_modified=yesterday,
-    ):
-        ack_file_set.add(key.lstrip("root/dispatch").upper().replace(".ACK.TXT", ""))  # noqa
-
-    # strip empty element before comparison
-    ack_file_set.discard("")
-    zip_file_set.discard("")
-
-    message = (
-        "Letter ack file does not contain all zip files sent. See runbook at "
-        "https://github.com/alphagov/notifications-manuals/wiki"
-        "/Support-Runbook#letter-ack-file-does-not-contain-all-zip-files-sent\n\n"
-        f"pdf bucket: {current_app.config['S3_BUCKET_LETTERS_PDF']}, "
-        f"subfolder: {datetime.utcnow().strftime('%Y-%m-%d')}/zips_sent\n"
-        f"ack bucket: {current_app.config['S3_BUCKET_DVLA_RESPONSE']}\n\n"
-        f"Missing ack for zip files: {str(sorted(zip_file_set - ack_file_set))}"
-    )
-
-    if len(zip_file_set - ack_file_set) > 0:
-        if current_app.should_send_zendesk_alerts:
-            ticket = NotifySupportTicket(
-                subject="Letter acknowledge error",
-                message=message,
-                ticket_type=NotifySupportTicket.TYPE_INCIDENT,
-                notify_ticket_type=NotifyTicketType.TECHNICAL,
-                ticket_categories=["notify_letters"],
-            )
-            zendesk_client.send_ticket_to_zendesk(ticket)
-
-        current_app.logger.error(
-            "Letter ack file does not contain all zip files sent.",
-            extra=dict(
-                pdf_bucket=current_app.config["S3_BUCKET_LETTERS_PDF"],
-                subfolder=datetime.utcnow().strftime("%Y-%m-%d") + "/zips_sent",
-                ack_bucket=current_app.config["S3_BUCKET_DVLA_RESPONSE"],
```

Comment:
I did a search for `S3_BUCKET_DVLA_RESPONSE` in our code. I can see we use it in this file but also in research_mode_tasks.py in `create_fake_letter_response_file`... 

I could approve this PR as is... but if you wouldn't mind also seeing if you could tidy up `create_fake_letter_response_file` too as I think we might be able to get rid of it? It is doing some fun stuff where it creates fake letter response files in dev and preview and I'm not sure we want that anymore at all...

That might enable us to remove the `S3_BUCKET_DVLA_RESPONSE` key from our config entirely

Reply:
I think we do still need the `create_fake_letter_response_file` task, because the `S3_BUCKET_DVLA_RESPONSE` bucket is / was used for two things.

1. It was being used for ack files when we sent letters using FTP (the code that's being deleted here).
2. It is still being used for response files. These get put in the bucket via SNS, so the code isn't in this repo apart from when we want to fake that in preview and dev using `create_fake_letter_response_file`

Reply:
Oh yes, good point. Sorry for my confusion.

Let me have a proper look at this PR and will approve shortly